Doc/faq/programming.rst

```diff
@@ -549,12 +549,12 @@ desired effect in a number of ways.
 
       args = {'a': 'old-value', 'b': 99}
       func3(args)
-      print(args['a'], args['b'])
+      print(args['a'], args['b'])     # output: new-value 100
```

Comment:
Would it be better to convert the snippet to console style?  Meaning

```python
>>> func3(args['a'], args['b'])
>>> print(args)
['new-value', 100]
```

Reply:
Should I change to console sytle?

Reply:
That’s the question I asked.  What do you and @tirkarthi think?

Reply:
Well I think we should change to console style

Reply:
I don't have a strong opinion on this. Other example codes have the output as comment like https://github.com/python/cpython/pull/21346/files#diff-986275b975a33c44c0aba973362516faL528 . So I am fine with either of the approach.

Reply:
Personally I don’t find output as comment very readable.
Let’s change it!